runtime/mainnet/src/lib.rs

```diff
@@ -435,12 +457,14 @@ construct_runtime!(
 		Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>},
 		TransactionPayment: pallet_transaction_payment::{Module, Storage},
 		Sudo: pallet_sudo::{Module, Call, Config<T>, Storage, Event<T>},
-		GenericAsset: pallet_generic_asset::{Module, Call, Storage, Config<T>, Event<T>},
+		CustomAsset: polkadex_custom_assets::{Module, Call, Storage, Config<T>, Event<T>},
 		Staking: pallet_staking::{Module, Call, Config<T>, Storage, Event<T>, ValidateUnsigned},
 		Session: pallet_session::{Module, Call, Storage, Event, Config<T>},
 		Historical: pallet_session_historical::{Module},
 		// Include the custom logic from the polkadex pallet in the runtime.
 		Polkadex: polkadex::{Module, Call, Storage, Event<T>},
+		PolkadexUniswap: polkadex_swap_engine::{Module, Call, Storage, Event<T>},
+        PolkadexIdentity: pallet_idenity::{Module, Call, Storage, Event<T>},
```

Comment:
Please correct the formatting here

Reply:
It's not showing on IDE. 
![Screenshot from 2021-01-13 17-04-12](https://user-images.githubusercontent.com/40355376/104447208-7c74a380-55c1-11eb-8bf3-fff5ba45e76e.png)